### PR TITLE
Changes list of headshot hosts

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request

Removes discord from viable headshot image sources.
Adds Imgbox, attributed to https://github.com/Bubberstation/Bubberstation/pull/1299
Fixes a bug that I made that didn't list Lensdump as an image host ingame

## How This Contributes To The Skyrat Roleplay Experience
Sourcing images from discord no longer works, as images are dropped after a few days, since Discord no longer wants to incidentally be an image hosting service.
Adds Imgbox as another host instead.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
</details>

## Changelog

:cl:
image: Headshots will no longer accept links from Discord, as they eventually break. Imgbox has been added as an alternative image host.
/:cl:

### CODE REVIEW

**Comment by lessthnthree** [comment]
If you're going to port, please include the changes to TGUI and player feedback as seen in https://github.com/Bubberstation/Bubberstation/pull/1299/files

**Comment by YakumoChen** [comment]
I've done it, but i can't include the bubber-specific parts obviously

**Comment by IgiariValkyr** [comment]
Can we get catbox.moe too? 👉👈

**Comment by lessthnthree** [comment]
> Can we get catbox.moe too? 👉👈

nya

**Comment by lessthnthree** [comment]
Agree on catbox, nya

File: modular_skyrat/master_files/code/modules/client/preferences/headshot.dm [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	maximum_value_length = MAX_MESSAGE_LEN
 	/// Assoc list of ckeys and their link, used to cut down on chat spam
 	var/list/stored_link = list()
-	var/static/link_regex = regex("i.gyazo.com|media.discordapp.net|cdn.discordapp.com|a.l3n.co|b.l3n.co|c.l3n.co") //gyazo, discord, lensdump
+	var/static/link_regex = regex("i.gyazo.com|a.l3n.co|b.l3n.co|c.l3n.co|images2.imgbox.com|thumbs2.imgbox.com") //gyazo, discord, lensdump, imgbox
```
Comment (Useroth) [inline modular_skyrat/master_files/code/modules/client/preferences/headshot.dm]:
```suggestion
	var/static/link_regex = regex("i.gyazo.com|a.l3n.co|b.l3n.co|c.l3n.co|images2.imgbox.com|thumbs2.imgbox.com|files.catbox.moe") //gyazo, discord, lensdump, imgbox, catbox
```

File: modular_skyrat/master_files/code/modules/client/preferences/headshot.dm [hunk 2]
```diff
@@ -36,7 +36,7 @@
 
 	find_index = findtext(value, link_regex)
 	if(find_index != 9)
-		to_chat(usr, span_warning("The image must be hosted on one of the following sites: 'Gyazo, Discord'"))
+		to_chat(usr, span_warning("The image must be hosted on one of the following sites: 'Gyazo, Lensdump, Imgbox'"))
```
Comment (Useroth) [inline modular_skyrat/master_files/code/modules/client/preferences/headshot.dm]:
```suggestion
		to_chat(usr, span_warning("The image must be hosted on one of the following sites: 'Gyazo, Lensdump, Imgbox, Catbox'"))
```

**Pull request merged**